yjit/src/backend/ir.rs

```diff
@@ -1724,7 +1724,9 @@ impl Assembler {
     }
 
     pub fn comment(&mut self, text: &str) {
-        self.push_insn(Insn::Comment(text.to_string()));
+        if cfg!(feature = "disasm") {
+            self.push_insn(Insn::Comment(text.to_string()));
```

Comment:
You might want to remove the `comment` function entirely so that folks are forced into using `asm_comment!`. You could replace the `$asm.comment(&format!($($fmt)*));` with `self.push_insn(Insn::Comment(text.to_string()));`

Reply:
Not a bad idea 👍